pkg/asset/cluster/aws/aws.go

```diff
@@ -55,27 +55,45 @@ func tagSharedVPCResources(ctx context.Context, clusterID string, installConfig
 		return err
 	}
 
-	ids := make([]*string, 0, len(privateSubnets)+len(publicSubnets))
-	for id := range privateSubnets {
-		ids = append(ids, aws.String(id))
-	}
-	for id := range publicSubnets {
-		ids = append(ids, aws.String(id))
-	}
-
 	session, err := installConfig.AWS.Session(ctx)
 	if err != nil {
 		return errors.Wrap(err, "could not create AWS session")
 	}
+	ec2Client := ec2.New(session, aws.NewConfig().WithRegion(installConfig.Config.Platform.AWS.Region))
+
+	ids := make([]*string, 0, len(publicSubnets))
+	for id := range publicSubnets {
+		ids = append(ids, aws.String(id))
+	}
 
+	// Add tags for Public Subnets
 	tagKey, tagValue := sharedTag(clusterID)
+	publicElbTagKey, publicElbTagValue := publicElbTag()
 
-	ec2Client := ec2.New(session, aws.NewConfig().WithRegion(installConfig.Config.Platform.AWS.Region))
 	if _, err = ec2Client.CreateTagsWithContext(ctx, &ec2.CreateTagsInput{
 		Resources: ids,
-		Tags:      []*ec2.Tag{{Key: &tagKey, Value: &tagValue}},
+		Tags: []*ec2.Tag{
+			{Key: &tagKey, Value: &tagValue},
+			{Key: &publicElbTagKey, Value: &publicElbTagValue},
+		},
 	}); err != nil {
-		return errors.Wrap(err, "could not add tags to subnets")
+		return errors.Wrap(err, "could not add tags to public subnets")
+	}
+
+	ids = make([]*string, 0, len(privateSubnets))
+	for id := range privateSubnets {
+		ids = append(ids, aws.String(id))
+	}
+	// Add tags for Private Subnets
+	privateElbTagKey, privateElbTagValue := privateElbTag()
```

Comment:
As this is the only place privateElbTag() is used, perhaps these variables could be simplified by statically assigning the values here.

Reply:
Using a static string or a constant did not work because Tag and Value expect *string and the conversion is not automatic.

Reply:
Done.